performance_regression_cdc_test.py

```diff
@@ -0,0 +1,229 @@
+
+        node1: BaseNode = self.db_cluster.nodes[0]
+
+        self.truncate_base_table(node1)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true}}")
+
+        self.wait_no_compactions_running()
+        self.run_fstrim_on_all_db_nodes()
+
+        self._workload_cdc(write_cmd,
+                           stress_num=2,
+                           test_name="test_write",
+                           sub_type="cdc_enabled")
+
+        self.wait_no_compactions_running()
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_write_with_cdc_preimage(self):
+        write_cmd = self.params.get("stress_cmd_w")
+
+        self._workload_cdc(write_cmd,
+                           stress_num=2,
+                           test_name="test_write",
+                           sub_type="cdc_disabled")
+        node1: BaseNode = self.db_cluster.nodes[0]
+
+        self.truncate_base_table(node1, cdclog_table=True)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'preimage': true}}")
+
+        self._workload_cdc(write_cmd,
+                           stress_num=2,
+                           test_name="test_write",
+                           sub_type="cdc_preimage_enabled")
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_write_with_cdc_postimage(self):
+        write_cmd = self.params.get("stress_cmd_w")
+
+        self._workload_cdc(write_cmd,
+                           stress_num=2,  # pylint: disable=unused-variable
+                           test_name="test_write",
+                           sub_type="cdc_disabled")
+        node1: BaseNode = self.db_cluster.nodes[0]
+
+        self.truncate_base_table(node1, cdclog_table=True)
+        node1.run_cqlsh(f"ALTER TABLE {self.keyspace}.{self.table} WITH cdc = {{'enabled': true, 'postimage': true}}")
+
+        self._workload_cdc(write_cmd,
+                           stress_num=2,  # pylint: disable=unused-variable
+                           test_name="test_write",
+                           sub_type="cdc_preimage_enabled")
+
+        self.check_regression_with_baseline(base_line="cdc_disabled")
+
+    def test_write_throughput(self):
+        self.cdc_workflow()
+
+    def test_write_latency(self):
+        self.cdc_workflow()
+
+    def test_mixed_throughput(self):
+        self.cdc_workflow(use_cdclog_reader=True)
+
+    def test_mixed_latency(self):
+        self.cdc_workflow(use_cdclog_reader=True)
+
+    def cdc_workflow(self, use_cdclog_reader=False):  # pylint: disable=unused-variable
+        self.keyspace = "keyspace1"
+        self.table = "standard1"
+        write_cmd = self.params.get("stress_cmd_w")
+        update_es = None
+
+        if use_cdclog_reader:
+            self.cdclog_reader_cmd = self.params.get('stress_cdclog_reader_cmd')
+            update_es = self.params.get('update_cdclog_reader_stats', default=False)
```

Comment:
please remove defaults from code, they should be in `test_defaults.yaml`

Reply:
moved